dbt_coves/utils/tracking.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from dbt import version as dbt_version
+from mixpanel import Mixpanel
+
+from dbt_coves import __version__ as dbt_coves_version
+
+MIXPANEL_DEV_TOKEN = os.environ.get("MIXPANEL_DEV_TOKEN", "3ba306217d298cc3b1a1ffe3442e938c")
+MIXPANEL_PROD_TOKEN = os.environ.get("MIXPANEL_PROD_TOKEN", "602e75af80a5c6d103c83bd2c675b247")
+
+
+def trackable(task, **kwargs):
+    def wrapper(task_instance, **kwargs):
+        mixpanel = Mixpanel(token=_get_mixpanel_env_token())
```

Comment:
This should be defined globally so the instance is reused. Move it out of the decorator function.